src/components/FilmList/Films.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import './Films.scss';
+import { isEmptyObject, isNonEmptyArray } from './deps';
+import { ListElement } from './type';
+
+class Films extends React.Component<ListElement> {
+    state = {
+        isShow: false,
+    };
+    onClick = (): void => {
+        this.setState({

```

Comment:
It's better to use "prevState".
`
this.setState(prevState => ({ isShow: !prevState.isShow }));
`